### DISCUSSION TITLE
No permissions to create pull request

### DISCUSSION THREAD

**Discussion by spirin** [stretchr/testify#1641] [General]
Hi! I can't make pull request:

```
git push -u origin unexpected-method-caller-info [1848ms]
2024-09-27 11:07:54.705 [info] ERROR: Permission to stretchr/testify.git denied to spirin.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Did I miss something?

**Comment by brackendawson** [comment]
You are not allowed to push to our repo. See the GitHub docs on creating a pull request from a fork: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by spirin** [comment]
Ok, thanks
